static/app/components/events/interfaces/llm-monitoring/llmMonitoringSection.tsx

```diff
@@ -2,38 +2,15 @@ import {LinkButton} from 'sentry/components/core/button';
 import {ButtonBar} from 'sentry/components/core/button/buttonBar';
 import {IconOpen} from 'sentry/icons';
 import {t} from 'sentry/locale';
-import type {Event} from 'sentry/types/event';
-import {MutableSearch} from 'sentry/utils/tokenizeSearch';
 import * as ModuleLayout from 'sentry/views/insights/common/components/moduleLayout';
-import {useSpansIndexed} from 'sentry/views/insights/common/queries/useDiscover';
+import LlmEventNumberOfPipelinesChartWidget from 'sentry/views/insights/common/components/widgets/llmEventNumberOfPipelinesChartWidget';
+import LlmEventTotalTokensUsedChartWidget from 'sentry/views/insights/common/components/widgets/llmEventTotalTokensUsedChartWidget';
 import {useModuleURL} from 'sentry/views/insights/common/utils/useModuleURL';
-import {
-  NumberOfPipelinesChart,
-  TotalTokensUsedChart,
-} from 'sentry/views/insights/llmMonitoring/components/charts/llmMonitoringCharts';
-import {SpanIndexedField} from 'sentry/views/insights/types';
 import {SectionKey} from 'sentry/views/issueDetails/streamline/context';
 import {InterimSection} from 'sentry/views/issueDetails/streamline/interimSection';
 
-interface Props {
-  event: Event;
-}
-
-export default function LLMMonitoringSection({event}: Props) {
+export default function LLMMonitoringSection() {
   const moduleUrl = useModuleURL('ai');
-  const trace = event.contexts.trace;
-
-  const {data} = useSpansIndexed(
-    {
-      limit: 1,
-      fields: [SpanIndexedField.SPAN_AI_PIPELINE_GROUP],
-      search: new MutableSearch(`trace:${trace?.trace_id} id:"${trace?.span_id}"`),
-      enabled: Boolean(trace?.span_id) && Boolean(trace?.trace_id),
-    },
-    'api.ai-pipelines.view'
-  );
-
-  const aiPipelineGroup = data[0]?.[SpanIndexedField.SPAN_AI_PIPELINE_GROUP];
```

Comment:
Moved into [a new hook](https://github.com/getsentry/sentry/pull/90094/files#diff-d3ed8fe4c16330df80d432ce45c6e7c9ed652bf0f2f4711b425b756b68a649ec) that is used in the new [widgets](https://github.com/getsentry/sentry/pull/90094/files#diff-498d040ecba70710587c3e02c8a57b236127f016740715b1dea0968bee98e0be)